appintro/src/main/res/values-ru/strings.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_intro_skip_button">Пропустить</string>
-    <string name="app_intro_done_button">Готово</string>
+    <string name="app_intro_skip_button">ПРОПУСТИТЬ</string>
+    <string name="app_intro_next_button">ДАЛЕЕ</string>
+    <string name="app_intro_back_button">НАЗАД</string>
+    <string name="app_intro_done_button">ГОТОВО</string>
```

Comment:
Can I ask you to add also this translation for the content description of the image ?
```
    <string name="app_intro_image_content_description">[Translation for "graphics"]</string>
```

Reply:
Done